tests/BaseTest.php

```diff
@@ -171,12 +172,22 @@ protected function getAuditManager(): AuditManager
         $auditConfig->setGlobalIgnoreColumns(['ignoreme']);
         $auditConfig->setUsernameCallable(static fn (): string => 'beberlei');
 
-        $auditManager = new AuditManager($auditConfig);
+        $auditManager = new AuditManager($auditConfig, $this->getClock());
         $auditManager->registerEvents($this->_getConnection()->getEventManager());
 
         return $this->auditManager = $auditManager;
     }
 
+    protected function getClock(): ClockInterface
```

Comment:
This should return null by default instead to test that we correctly work without clock.
```suggestion
    protected function getClock(): ?ClockInterface
    {
         return null;
    }
```

Reply:
Done